README.md

```diff
@@ -44,6 +46,15 @@ permitted_domains: 'hashrocket.com|hshrckt.com'
 permitted_emails: 'friend@whitelist.com'
 ```
 
+##### To get admin route working on localhost:
+
+```
+1. Navigate to https://console.developers.google.com/
```

Comment:
Interesting idea. Could we link to the Google Oauth README or a Google article instead? It seems like any of these steps could change without us knowing, potentially leading developers down a dead-end.